rules-reviewed/technology-usage/database-technology-usage.windup.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0"?>
+<ruleset id="technology-usage-database" xmlns="http://windup.jboss.org/schema/jboss-ruleset" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides statistical summaries of the database items that were found during the analysis.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.7.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.7.0.Final" />
+        </dependencies>
+        <phase>PostMigrationRulesPhase</phase>
+    </metadata>
+    <rules>
+        <rule id="technology-usage-database-01000">
+            <when>
+                <graph-query discriminator="JdbcDatasourceModel"  />
```

Comment:
I haven't been able to find a rule that adds `JdbcDatasourceModel` to the graph so this rule was always failing its test.
I think that it might be better to switch to use `DataSourceModel` instead.

Reply:
You are right. I thought there is some relation between `DataSourceModel` and `JdbcDataSourceModel`, but it seems there isn't.

I was quite confused about this too and didn't solve it yet. I guess using just `DataSource` would be better...

Reply:
yep, i think that using `DataSource` will be good for this rule.
Maybe we can also think about removing `JdbcDatasourceModel` because it's quite useless right now.

Reply:
I created JIRA for that. I  will eventually get back to it and remove it